mutiny-wasm/Cargo.toml

```diff
@@ -43,6 +43,7 @@ urlencoding = "2.1.2"
 once_cell = "1.18.0"
 payjoin = { version = "0.13.0", features = ["send", "base64"] }
 fedimint-core = { git = "https://github.com/fedimint/fedimint", rev = "6a923ee10c3a578cd835044e3fdd94aa5123735a" }
+moksha-core = { git = "https://github.com/ngutech21/moksha" }
```

Comment:
we should pin these to a specific commit so stuff doesn't break, can you add `rev = "0b9a32d216ba0782a4aa2cdb19a87589e16d9089"`